src/util-inl.h

```diff
@@ -198,6 +198,18 @@ TypeName* Unwrap(v8::Local<v8::Object> object) {
   return static_cast<TypeName*>(pointer);
 }
 
+void SwapBytes(uint16_t* dst, const uint16_t* src, size_t buflen) {
+  for (size_t i = 0; i < buflen; i++) {
+      #ifdef __GNUC__
+        dst[i] = __builtin_bswap16(src[i]);
```

Comment:
There's really no need for this.  Check the generated machine code with and without the intrinsic.